include/lsst/afw/geom/Transform.h

```diff
@@ -198,9 +198,9 @@ class Transform {
      *          this Transform to the result. Its inverse shall first apply the
      *          inverse of this Transform, then the inverse of `first`.
      *
-     * @throws InvalidParameterErrror Thrown if `first.getToEndpoint()` and
-     *                                `this->getFromEndpoint()` do not have
-     *                                the same number of axes.
+     * @throws pex::exceptions::InvalidParameterErrror Thrown if
+     *         `first.getToEndpoint()` and `this->getFromEndpoint()` do not
+     *         have the same number of axes.
```

Comment:
Good catch. I unfortunately can't test this change locally because Doxygen doesn't have any `pex::exceptions` docs to link to.